### PR TITLE
Ee2

### PR DESCRIPTION
**Pull request by bio-boris**

### CODE REVIEW

File: kbase.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.4.3
+    0.4.6
```
Comment (jayrbolton) [inline kbase.yml]: why 0.4.3 to 0.4.6

Also maybe this is an opportunity to add a changelog? I like to use https://keepachangelog.com/en/1.0.0/

File: lib/KBParallel/utils/job.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
-from installed_clients.NarrativeJobServiceClient import NarrativeJobService
 from KBParallel.baseclient import BaseClient
 from KBParallel.baseclient import ServerError
 
+from installed_clients.execution_engine2Client import execution_engine2
```
Comment (jayrbolton) [inline lib/KBParallel/utils/job.py]: is `execution_engine2` a classname? If so, can it be camel uppercase like other classes?

File: kbase.yml (hunk 1 above)
Comment (bio-boris) [reply kbase.yml]: I think it had something to do with catalog registration not working

File: lib/KBParallel/utils/job.py [hunk 3]
```diff
@@ -85,12 +92,13 @@ def check_local_status(self):
...
-            'job_ids': [self.job_id],
-            'with_job_params': 0
-        })
-        job_state = response['job_states'][self.job_id]
+
+        cjp = {'job_id': self.job_id, 'projection': []}
+        response = self.ee2.check_job(params=cjp)
+        from pprint import pprint
+        pprint(response)
+        job_state = response
```
Comment (jayrbolton) [inline lib/KBParallel/utils/job.py]: no reason to have this additional assignment here

File: lib/KBParallel/utils/job.py (hunk 2 above)
Comment (bio-boris) [reply lib/KBParallel/utils/job.py]: That's the name of the client and class. Are you saying to import it as "ExecutionEngine2" or "EE2" or something?

File: lib/installed_clients/execution_engine2Client.py [hunk 4]
```diff
@@ -0,0 +1,851 @@
...
+# package or not. This makes pep8 unhappy hence the annotations.
+try:
+    # baseclient and this client are in a package
+    from .baseclient import BaseClient as _BaseClient  # @UnusedImport
+except ImportError:
+    # no they aren't
+    from baseclient import BaseClient as _BaseClient  # @Reimport
+
+
+class execution_engine2(object):
```
Comment (jayrbolton) [inline lib/installed_clients/execution_engine2Client.py:20]: can this be upper camel case?

File: lib/KBParallel/utils/job.py (hunk 3 above)
Comment (bio-boris) [reply lib/KBParallel/utils/job.py]: fixed

**Pull request merged**